main.js

```diff
@@ -0,0 +1,29 @@
+    var CarLot = (function() {
+
+
+        function populatePage(inventory) {
+            // Loop over the inventory and populate the page
+
+
+document.getElementById("text").addEventListener("keyup", carBoundToText);
+
+function carBoundToText(e) {
```

Comment:
Is this function doing anything in this file? It looks like you might be doing it in your print.js instead.